### PR TITLE
Create Uptime SLO

### PR DESCRIPTION
**Pull request by JonPHardy57**
Add in an Octopus Cloud page to document the Octopus Cloud SLO and the basis for interpretation.

### CODE REVIEW

**Comment by JonPHardy57** [comment]
Adding in a page to document the Octopus Cloud SLO.

File: docs/octopus-cloud/Uptime SLO [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Octopus Cloud Uptime SLO
+position: 50
+description: The uptime SLO for Octopus Cloud instances
+---
+
+## Octopus Cloud Uptime
```
Comment (harrisonmeister) [inline docs/octopus-cloud/Uptime SLO:7]: I don't think a H2 heading is required as there is no preceding paragraph. This results in a H1 stating **Octopus Cloud Uptime SLO**, directly followed by a H2 with **Octopus Cloud Uptime**. An alternative here would be to add a preceding paragraph stating what the page is for.
```suggestion
```

File: docs/octopus-cloud/Uptime SLO [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Each Octopus Deploy Cloud customer has their own instance of the Octopus Server as well as a Dynamic Worker. As the name implies the workers are dynamic and are spun up and down as required by the Deployment or Runbook executed. The following uptime SLO (service level objective) therefore refers to the customer server instances. 
+
+Each customer’s instance may experience its own series of maintenance operations and reprovisioning for operational and upgrade reasons. Therefore a 95th percentile of monthly uptime is used as the basis for the Octopus Cloud uptime SLO. Operational downtime is, other than in exceptional circumstances, scheduled in the customer’s maintenance window. All downtime (unplanned and planned) is used in the determination of the uptime SLO.
+
+### Uptime SLO
```
Comment (harrisonmeister) [inline docs/octopus-cloud/Uptime SLO:12]: See H2 heading comment at the start of this page
```suggestion
## Uptime SLO
```

File: docs/octopus-cloud/Uptime SLO [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+title: Octopus Cloud Uptime SLO
+position: 50
+description: The uptime SLO for Octopus Cloud instances
+---
+
+## Octopus Cloud Uptime
+Each Octopus Deploy Cloud customer has their own instance of the Octopus Server as well as a Dynamic Worker. As the name implies the workers are dynamic and are spun up and down as required by the Deployment or Runbook executed. The following uptime SLO (service level objective) therefore refers to the customer server instances.
```
Comment (harrisonmeister) [inline docs/octopus-cloud/Uptime SLO:8]:
```suggestion
Each Octopus Cloud customer has their own instance of the Octopus Server and can use [dynamic workers](/docs/infrastructure/workers/dynamic-worker-pools.md). As the name implies, these workers are assigned to a cloud instance dynamically and are spun up and down as required by the Deployment or Runbook executed. The following uptime SLO (service level objective), therefore, refers to the customer's Cloud instance.
```

File: docs/octopus-cloud/Uptime SLO [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+title: Octopus Cloud Uptime SLO
+position: 50
+description: The uptime SLO for Octopus Cloud instances
+---
+
+## Octopus Cloud Uptime
+Each Octopus Deploy Cloud customer has their own instance of the Octopus Server as well as a Dynamic Worker. As the name implies the workers are dynamic and are spun up and down as required by the Deployment or Runbook executed. The following uptime SLO (service level objective) therefore refers to the customer server instances. 
+
+Each customer’s instance may experience its own series of maintenance operations and reprovisioning for operational and upgrade reasons. Therefore a 95th percentile of monthly uptime is used as the basis for the Octopus Cloud uptime SLO. Operational downtime is, other than in exceptional circumstances, scheduled in the customer’s maintenance window. All downtime (unplanned and planned) is used in the determination of the uptime SLO.
```
Comment (harrisonmeister) [inline docs/octopus-cloud/Uptime SLO:10]:
```suggestion
Each customer's instance may experience its own series of maintenance operations and reprovisioning for operational and upgrade reasons. Therefore the 95th percentile of monthly uptime is used as the basis for the Octopus Cloud uptime SLO. Operational downtime is, other than in exceptional circumstances, scheduled in the customer’s [maintenance window](/docs/octopus-cloud/index.md#set-the-outage-window). All downtime (unplanned and planned) is used in the determination of the uptime SLO.
```

File: docs/octopus-cloud/Uptime SLO [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
```
Comment (harrisonmeister) [inline docs/octopus-cloud/Uptime SLO:1]: this page will need to end in .md e.g. `uptime-slo.md`

**Review by harrisonmeister — COMMENTED** [review]
Build failing due to page naming I suspect. Made some other minor suggestions

File: docs/octopus-cloud/Uptime SLO (hunk 5 above)
Comment (JonPHardy57) [reply docs/octopus-cloud/Uptime SLO]: Done, thank you for your patience. I am sure you have guessed this is my first-time using GitHub as I am not a developer.

Comment (harrisonmeister) [reply docs/octopus-cloud/Uptime SLO]: No worries, I updated the page again to lower case and removed spaces :)

**Review by harrisonmeister — APPROVED** [review]
LGTM

**Pull request merged**